src/test/java/org/cactoos/io/TempFileTest.java

```diff
@@ -43,19 +43,19 @@
     @Test
     public void createFile() throws Exception {
         try (TempFile file = new TempFile()) {
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "Cannot create a temp file",
                 Files.exists(file.value()),
                 Matchers.is(true)
```

Comment:
@marceloamadeu can you change that to `new IsTrue()`?

Reply:
@victornoel Done ... please check.